src/background/actions/dev-tools-action-creator.ts

```diff
@@ -39,7 +41,8 @@ export class DevToolsActionCreator {
         );
     }
 
-    private onDevToolOpened = (): void => {
+    private onDevToolOpened = (_: unknown, tabId: number): void => {
+        this.devToolsMonitor.startMonitoringDevtool(tabId);
```

Comment:
This feels a little off; I think the more typical pattern would be for the monitor to register as a `devToolActions.setDevToolState` listener, rather than hardcoding knowledge of the monitor into the action creator.